lms/urls.py

```diff
@@ -61,6 +61,9 @@
 
     url(r'^heartbeat$', include('heartbeat.urls')),
 
+    # TODO jimabramson: put this in the right place
+    url(r'^api/profile_images/', include('openedx.core.djangoapps.profile_images.urls')),
```

Comment:
That's an interesting question. APIs are meant to be one-to-one with their Django app, so the way it is written it makes sense to be a different API. I think part of the issue is having such a specific feature be a top level API. Maybe this should be an end point of a more general 'file upload' API. If it is just about profile images, then I agree with @cahrens that it probably should just be an end point in the User API.